111 reviews Styling carousel of Customer Reviews

vricci518 opened the pull request:
Closes #111 
Stylize a carousel of customer reviews
Need to link to the database to pull data and populate fields

vricci518 committed:
Initial styling of Customer Reviews section

vricci518 committed:
Styling of Customer Reviews carousel

billythakidd04 commented:
@vricci518 this should utilize the api. At no point should it concern itself with the db. We work with the data that the api gives us. If something is missing from the api, that needs to be dealt with separately. Very important distinction.

vricci518 committed:
Updated testimonial slider and css styling

vricci518 committed:
Mergg branch 'master' into 111-reviews
 ter into workin branlease enter a commit message to explain why this merge is necessary,
:wq

vricci518 committed:
merged updated master into working file

vricci518 committed:
Updated styling on testimonials

vricci518 committed:
Fixed conflicts between Specials Carousel and Reviews Carousel

vricci518 commented:
Created a carousel of reviews that are pulled from the reviewsAPI.
Needed to add a class called hero-specials to the Hero Image and the Specials Carousel to fix conflicts with the Reviews Carousel.
@billythakidd04 still cannot get data to populate from the reviewsAPI. Need assistance with that. I'm not requesting reviews from the team until we resolve this.

billythakidd04 commented:
@vricci518 call me when you can.

vricci518 committed:
Updated Review Carousel fixed conflict in Hero on About Apply pages

vricci518 committed:
Added score cups for half cups

vricci518 commented:
Loaded score cup images for when .5 scores are given. Carousel of reviews is working.
@billythakidd04 when the url is http://localhost:8888/starbulls/ the API is pulling the data. But when you select the logo to go back to the home page the url is http://localhost:8888/starbulls/index.php and the placeholder reviews show up rather than the API data.

eiman-kased commented:
> Loaded score cup images for when .5 scores are given. Carousel of reviews is working.
> @billythakidd04 when the url is http://localhost:8888/starbulls/ the API is pulling the data. But when you select the logo to go back to the home page the url is http://localhost:8888/starbulls/index.php and the placeholder reviews show up rather than the API data.

@vricci518 @billythakidd04 
Carousel looks amazing! 
Only the placeholder reviews show up regardless of the URL.

eiman-kased commented:
![Screen Shot 2021-04-19 at 4 21 02 PM](https://user-images.githubusercontent.com/78566785/115298390-6bd94980-a12b-11eb-8c97-c0a3d30e3c35.png)

The URL is missing a forward slash between the file name and the route.

billythakidd04 committed:
added rewrite rule for removing index.php from url for #110

billythakidd04 committed:
Merge branch '111-reviews' of ssh://github.com/eiman-kased/starbulls into 111-reviews

billythakidd04 committed:
fixed logo link caused by .htaccess update in a7e79a8 for #111

margeschrec reviewed (approved):
works well, I think we should have the review drop down below the form on smaller screen, an idea for a different ticket. 

![image](https://user-images.githubusercontent.com/78566744/115312127-c4fea880-a13e-11eb-89bb-f6a0c4213935.png)

nburd90 reviewed (approved):
![Screen Shot 2021-04-19 at 6 40 09 PM](https://user-images.githubusercontent.com/78566780/115312105-bc0dd700-a13e-11eb-88fc-69c799c3e604.png)

Minor responsive issue. Can create a separate ticket for this.
